training/src/main/java/wolox/training/controllers/ClientController.java

```diff
@@ -41,6 +45,7 @@ public Client addBook(@RequestBody Book book, @PathVariable Long id) {
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
     public Client create(@RequestBody Client client) {
+        client.setPassword(passwordEncoder.encode(client.getPassword()));
```

Comment:
This is not necessary, as the setter is already called by Jaxckson when creating the Client object, so it's already encoded :)